src/game_api/entities_floors.hpp

```diff
@@ -365,3 +389,13 @@ class PoleDeco : public Floor
     int32_t deco_up;
     int32_t deco_down;
 };
+
+class JungleSpearTrap : public Floor
+{
+  public:
+    /// The uid must be movable entity for ownership transfers, direction: 1 = left, 2 = right, 3 = up, 4 = down
```

Comment:
Maybe an enum instead?

Reply:
If i could make it also for something else, but making global enum just for this one function feels wasteful